src/apps/competitions/emails.py

```diff
@@ -0,0 +1,90 @@
+from utils.email import codalab_send_mail
```

Comment:
I really like the idea of this!! Makes these nice and easy to test if we ever want to just hit the template or some-such